.gitattributes

```diff
@@ -31,3 +31,6 @@
 src/e2e/resources/data/*            linguist-vendored
 src/test/resources/data/*           linguist-vendored
 src/test/resources/emails/*         linguist-vendored
+
+# docker scripts should have LF line-endings, even if checked out on windows:
```

Comment:
The comment doesn't match the implementation: `Docker scripts` and `*.sh` are not the same, whereby the latter is obviously all shell scripts.

Reply:
Fair point, the only other case is the 'wait-for-server' shell script, which wouldn't work on windows anyway. (both contain shebangs)

I wanted to include that What, and the Why, and my attempt at brevity cost a bit of truth.

Windows scripts tend to end in .bat, .cmd or .ps1 so I'll edit the comment.

The alternative is to deliberately target the solr/*.sh and datastore/*.sh container folders, but I don't think people will remember to update the .gitattributes accordingly if someone renames the containers.